### PR TITLE
Prepare 2.0.3

### PR DESCRIPTION
**Pull request by ralphtheninja**
This PR should not be merged. Only for review.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,18 @@
 
 ## [Unreleased]
 
+## [2.0.3] - 2018-06-28
+
+### Changed
+* Reverted proper destroy (#34) (@ralphtheninja)
+
+**Historical Note** @ralphtheninja was trigger happy and released `v2.0.2` incorrectly, which should only have included a fix to node 4.
```
Comment (vweevers) [inline CHANGELOG.md]: No need to take the blame (we all love you to bits), IMO just say "**Historical Note** The previous release was meant to restore node 4 and included an additional change by mistake."

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,18 @@
 
 ## [Unreleased]
 
+## [2.0.3] - 2018-06-28
+
+### Changed
```
Comment (vweevers) [inline CHANGELOG.md]: Changed -> Fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,18 @@
...
+* Reverted proper destroy (#34) (@ralphtheninja)
```
Comment (vweevers) [inline CHANGELOG.md]: Reverted -> Revert

**Comment by ralphtheninja** [comment]
Removing branch, since we have the tag.

**Pull request closed without merging**